modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/app/App.js

```diff
@@ -185,6 +185,15 @@ class App extends EventEmitter {
 		 */
 		this.popstateScrollTop = 0;
 
+		/**
+		 * Whether to preload CSS files prior to surface flipping so that FOUC
+		 * does not happen.
+		 * @type {!Boolean}
+		 * @default false
+		 * @protected
+		 */
+		this.preloadCSS = !!config?.preloadCSS;
```

Comment:
Why `!!`? Is `config.preloadCSS` something other than a boolean? If it is, can it be string `'false'`? Because `!!'false'` is `true`.

Reply:
It is `undefined` in unit tests (don't you love JavaScript :heart_eyes: ?) Regarding `!!` it's the common idiom to make truthy/falsy things true/false. 

It's true that `'false'` yields true, as you say, but that's expected and works like other languages too (more or less... :roll_eyes:  ).

Reply:
Ah right, `undefined`. JS :hugs: